applications/zigbee_weather_station/README.rst

```diff
@@ -0,0 +1,519 @@
+            IEEE 802.15.4 Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Network Layer Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Application Support Layer Data, Dst Endpt: 64, Src Endpt: 42
+            ZigBee Cluster Library Frame, Command: Report Attributes, Seq: 128
+               Frame Control Field: Profile-wide (0x08)
+               Sequence Number: 128
+               Command: Report Attributes (0x0a)
+               Attribute Field
+                  Attribute: Measured Value (0x0000)
+                  Data Type: 16-Bit Signed Integer (0x29)
+                  Measured Value: 24.26 [°C]
+
+      * Pressure cluster periodic report will look like as follows:
+
+         .. code-block::
+
+            ZigBee HA	3888	7705.727682	0x1ca9	0x0000	0x1ca9	0x0000	f4:ce:36:23:d7:4e:77:1d	f4:ce:36:55:fc:08:bf:6b		0xbf9e	ZCL: Report Attributes, Seq: 129
+
+            Frame 3888: 67 bytes on wire (536 bits), 67 bytes captured (536 bits)
+            IEEE 802.15.4 Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Network Layer Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Application Support Layer Data, Dst Endpt: 64, Src Endpt: 42
+            ZigBee Cluster Library Frame, Command: Report Attributes, Seq: 129
+               Frame Control Field: Profile-wide (0x08)
+               Sequence Number: 129
+               Command: Report Attributes (0x0a)
+               Attribute Field
+                  Attribute: Measured Value (0x0000)
+                  Data Type: 16-Bit Signed Integer (0x29)
+                  Measured Value: 98.4 [kPa]
+
+      * Relative humidity cluster periodic report will look like as follows:
+
+         .. code-block::
+
+            ZigBee HA	3894	7705.856763	0x1ca9	0x0000	0x1ca9	0x0000	f4:ce:36:23:d7:4e:77:1d	f4:ce:36:55:fc:08:bf:6b		0xbf9e	ZCL: Report Attributes, Seq: 130
+
+            Frame 3894: 67 bytes on wire (536 bits), 67 bytes captured (536 bits)
+            IEEE 802.15.4 Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Network Layer Data, Dst: 0x0000, Src: 0x1ca9
+            ZigBee Application Support Layer Data, Dst Endpt: 64, Src Endpt: 42
+            ZigBee Cluster Library Frame, Command: Report Attributes, Seq: 130
+               Frame Control Field: Profile-wide (0x08)
+               Sequence Number: 130
+               Command: Report Attributes (0x0a)
+               Attribute Field
+                  Attribute: Measured Value (0x0000)
+                  Data Type: 16-Bit Unsigned Integer (0x21)
+                  Measured Value: 32.99 [%]
+
+.. WIP: PLACEHOLDER
+
+   .. _zigbee_weather_station_app_dfu:
+
+   Updating the device firmware
+   ============================
+
+   * Placeholder for application FOTA procedure
+
+      To update the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial in the Matter documentation.
```

Comment:
```suggestion
      To update the device firmware, complete the following steps;
```
Let's remove the Matter refs even from placeholder content.

Reply:
Done